js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,14 @@
+function abbreviateString(str) {
+  if (typeof str !== 'string') {
+    throw new Error(`Expected a string, but received ${typeof str} type`);
+  }
+
+  const separatorRegex = /\s+/g;
+  const words = str.trim().split(separatorRegex);
+  const abbreviationFirstWord = words[0];
+  const abbreviationLastWord = `${words[words.length - 1][0].toUpperCase()}.`;
```

Comment:
it would be better,, just a info - abbreviationLastWord to abberviationLastWordChar , rest all are awesome, you have used const , and nice indentation.

Reply:
@vinodkumartheking I have used abbreviationLastWord because it consists of 2 characters i.e 1st letter of the last word and a dot. So grammatically its a word.

Let me know what do you think about it.

Reply:
@vinodkumartheking I did some searching and got to know grammatically correct word doesn't contain punctuation marks. So now I could also say `abberviationLastWordChar`  could be a better variable name.